packages/manager/src/features/Kubernetes/CreateCluster/ApplicationPlatform.tsx

```diff
@@ -49,19 +54,27 @@ export const ApplicationPlatform = (props: APLProps) => {
       >
         <Box alignItems="center" display="flex" flexDirection="row">
           <Typography data-testid="apl-label">Akamai App Platform</Typography>
-          <Chip color="primary" label="BETA" sx={{ ml: 1 }} />
+          <Chip
+            color="primary"
+            data-testid="apl-beta-chip"
+            label={CHIP_COPY}
```

Comment:
Can we make the chip text smaller?

| Medium  | Small   |
| ------- | ------- |
| ![image](https://github.com/user-attachments/assets/5855be0b-a74c-4c49-885c-a84b28a86408) | ![image](https://github.com/user-attachments/assets/2311cfa2-8db8-4d30-bc31-a2686e4a0b17) |

```suggestion
            label={CHIP_COPY}
            size="small"
```

Reply:
Done in [b5362b1](https://github.com/linode/manager/pull/11809/commits/b5362b1d9da472c8d66f2deff64c9ba8c39aea3c). Took a look through our other chips and confirmed the majority of them had small text, so followed suit here. Thanks for the suggestion.